### PR TITLE
Simplify README

### PR DESCRIPTION
**Pull request by johnboxall**
I've tried to simplify the README by removing uncommon usage patterns, linking out to more complete external resources and removing unneeded language and duplicate links.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,15 @@
 # commerce-sdk-isomorphic
 
-The Salesforce Commerce SDK (Isomorphic) allows easy interaction with the B2C Commerce platform’s Shopper APIs on the Node.js runtime and works both in browsers and Node applications. For a Node-based SDK that can access the Admin APIs in addition to the Shopper APIs, see the main [Commerce SDK](https://github.com/SalesforceCommerceCloud/commerce-sdk). 
+This SDK provides a Browser & Node.js JavaScript client for calling [B2C Commerce Shopper APIs](https://developer.salesforce.com/docs/commerce/commerce-api/overview).
```
Comment (johnboxall) [inline README.md:3]: Mention that this API is about SCAPI!

File: README.md [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
+
+_For a Node.js only SDK that can also access Admin APIs checkout [Commerce SDK](https://github.com/SalesforceCommerceCloud/commerce-sdk)._
 
 ## Getting Started
 
 ### Requirements
 
 - Node `^12.x`, `^14.x`, `^16.x`, `^18.x` or `^20.x`
-
+- The SDK requires B2C Commerce API (SCAPI) to be configured. For more info see [Getting started with SCAPI](https://developer.salesforce.com/docs/commerce/commerce-api/guide/get-started.html).
```
Comment (johnboxall) [inline README.md:12]: The pre-reqs for using SCAPI are described well here. We don't need to repeat this info in the README.

File: README.md [hunk 3]
```diff
@@ -17,55 +19,25 @@ npm install commerce-sdk-isomorphic
...
-| clientId       | ID of the client account created with Salesforce Commerce.                 |
-| organizationId | The unique identifier for your Salesforce identity.                        |
-| shortCode      | Region-specific merchant ID.                                               |
-| siteId         | Name of the site to access data from, for example, RefArch or SiteGenesis. |
-
-
-### Configure the Isomorphic SDK
-
-
 ```javascript
```
Comment (johnboxall) [inline README.md:22]: Simplified this code example, removing edge case for private client login and focusing on the specific calls.

File: README.md [hunk 4]
```diff
@@ -76,59 +48,33 @@ const searchResult = await shopperSearch.productSearch({
 });
 ```
 
-#### CORS
+#### Fetch Options
```
Comment (johnboxall) [inline README.md:51]: Focus on the `fetchOptions` rather than the entire rest of the config.

File: README.md [hunk 5]
```diff
@@ -76,59 +48,33 @@ const searchResult = await shopperSearch.productSearch({
...
 
-_throwOnBadResponse:_ Default value is false. When set to true, the SDK throws an Error on responses with statuses that are not 2xx or 304.
+#### Additional Config Settings
 
-### Public/Private Client Shopper Login (SLAS) helpers
-
-A collection of helper functions are available in this SDK to simplify [Public/Private Client Shopper Login OAuth flows](https://developer.salesforce.com/docs/commerce/commerce-api/references?meta=shopper-login:Summary). See sample code above for guest login.
-
-**⚠️ WARNING ⚠️**
-Users should be extremely cautious about using the SLAS private client helper functions in the browser as it can expose your client secret. Ensure that your client secret is secured before running the function client side.
```
Comment (joeluong-sfcc) [inline README.md:127]: I think we should consider leaving in this warning somewhere about SLAS private client and potentially exposing it on the client side

**Review by joeluong-sfcc — DISMISSED** [review]

File: README.md (hunk 5 above)
Comment (johnboxall) [reply README.md:127]: @joeluong-sfcc – I think using a Client Secret represents a very advanced use case that I wouldn't document in the README.

The function itself has this warning:

https://salesforcecommercecloud.github.io/commerce-sdk-isomorphic/modules/helpers.html#loginguestuserprivate-1

I don't feel strongly.

**Review by joeluong-sfcc — APPROVED** [review]

**Pull request merged**